Bottom: 1fc904d9906c82af7ef9108e89d666df8fb6d986
Top:    dbeb5563920c762a48751445d3e0aee2b5bd611b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 16:06:28 +0000

IMPORTANT: DONT DRAW POUR OUTLINES DURING EXPORT

They can produce shorted boards otherwise!


---

diff --git a/src/draw.c b/src/draw.c
index f6c714b..8ad94c5 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -708,7 +708,8 @@ pour_callback (const BoxType * b, void *cl)
   struct pin_info *i = (struct pin_info *) cl;
   PourType *pour = (PourType *)b;
 
-  DrawPour (i->Layer, pour, 0);
+  if (gui->gui)
+    DrawPour (i->Layer, pour, 0);
 
   if (pour->PolygonN)
     {
